internal/util-logging/src/main/scala/sbt/util/InterfaceUtil.scala

```diff
@@ -104,14 +120,26 @@ object InterfaceUtil {
   def problem(cat: String, pos: Position, msg: String, sev: Severity): Problem =
     problem(cat, pos, msg, sev, None)
 
+  @deprecated("Use the overload of this method with more arguments", "1.7.2")
   def problem(
       cat: String,
       pos: Position,
       msg: String,
       sev: Severity,
       rendered: Option[String]
   ): Problem =
-    new ConcreteProblem(cat, pos, msg, sev, rendered)
+    problem(cat, pos, msg, sev, None, None, List.empty[DiagnosticRelatedInformation])
```

Comment:
Is this throwing out rendered?

Reply:
O weird, yes you're correct. Good catch, I didn't mean to do this. I made this `rendered` is still captured now.